http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ZombieMonitorQueries.scala

```diff
@@ -3,41 +3,53 @@ package db
-    val joinedQuery = clusters.joinLeft(RuntimeConfigQueries.runtimeConfigs).on(_.runtimeConfigId === _.id)
-    joinedQuery.result.map { cs =>
-      cs.toList.foldMap { c =>
+  def listActiveZombieQuery(implicit ec: ExecutionContext): DBIO[Map[GoogleProject, Chain[ZombieCandidate]]] = {
+    val runtimes = clusterQuery.filter(_.status inSetBind RuntimeStatus.activeStatuses.map(_.toString))
+    val joinedQuery = runtimes.joinLeft(RuntimeConfigQueries.runtimeConfigs).on(_.runtimeConfigId === _.id)
+    joinedQuery.result.map { rs =>
+      rs.toList.foldMap { r =>
+        val clusterRec = r._1
+        // .get here should be okay since every cluster record in database should have a runtimeConfig
+        val cloudService = r._2.get.runtimeConfig.cloudService
+        val runtime = unmarshalCluster(clusterRec, Seq.empty, List.empty, Map.empty, List.empty, List.empty, List.empty, List.empty)
         Map(
-          c._1.googleProject -> Chain(
-            PotentialZombieRuntime(
-              c._1.id,
-              c._1.googleProject,
-              c._1.clusterName,
-              RuntimeStatus.withName(c._1.status),
-              c._1.auditInfo,
-              c._2.get.runtimeConfig.cloudService
-            ) // .get here should be okay since every cluster record in database should have a runtimeConfig
+          r._1.googleProject -> Chain(
+            ZombieCandidate(runtime, cloudService)
           )
         )
       }
     }
   }
 
+  private def unconfirmedRuntimeLabels: Query[LabelTable, LabelRecord, Seq] =
+    labelQuery.filter(_.key === Config.zombieClusterMonitorConfig.deletionConfirmationLabelKey).filter(_.value === "false")
+
+  def listInactiveZombieQuery(implicit ec: ExecutionContext): DBIO[List[ZombieCandidate]] = {
+    val runtimes = clusterQuery.filter(_.status === RuntimeStatus.Deleted.toString)
+    val unconfirmedRuntimeQuery = runtimes.join(unconfirmedRuntimeLabels).on(_.id === _.clusterId)
+    val joinedQuery = unconfirmedRuntimeQuery.joinLeft(RuntimeConfigQueries.runtimeConfigs).on(_._1.runtimeConfigId === _.id)
+    joinedQuery.result.map { rs =>
+      rs.toList.map { r =>
+        val clusterRec = r._1._1
+        // .get here should be okay since every cluster record in database should have a runtimeConfig
+        val cloudService = r._2.get.runtimeConfig.cloudService
+        val runtime = unmarshalCluster(clusterRec, Seq.empty, List.empty, Map.empty, List.empty, List.empty, List.empty, List.empty)
+        ZombieCandidate(runtime, cloudService)
+      }
+    }
+  }
 }
 
-final case class PotentialZombieRuntime(id: Long,
-                                        googleProject: GoogleProject,
-                                        runtimeName: RuntimeName,
-                                        status: RuntimeStatus,
-                                        auditInfo: AuditInfo,
-                                        cloudService: CloudService)
+final case class ZombieCandidate(runtime: Runtime,
```

Comment:
In order for the runtime to be deleted through a CloudService interpreter, we need to pass in the full runtime, which is why `ZombieCandidate` now has a full Runtime in it. The other option was to have two types of `ZombieCandidate`s - `ActiveZombieCandidate` and `InactiveZombieCandidate` but I thought it made the code in the monitor a lot less easily readable

Reply:
Delete really shouldn't need the whole `Runtime`...if you look at both gce and dataproc interpreters, they only really need googleProject, runtimeName, and zoneName (if gce)...Let's refactor `deleteRuntime` signature a bit? :)